sdk/core/azure-core-http-netty/README.md

```diff
@@ -22,6 +22,30 @@ Azure Core Netty HTTP client is a plugin for the azure-core HTTP client API.
 
 ## Examples
 
+The following sections provide several code snippets covering some of the most common client configuration scenarios.
+
+- [Create a Simple Client](#create-a-simple-client)
+- [Create a Client with Proxy](#create-a-client-with-proxy)
+
+### Create a Simple Client
+
+Create the simplest Netty Http client.
+
+```java
+HttpClient client = new NettyAsyncHttpClientBuilder()
+    .build();
```

Comment:
I always like seeing `.build()` on the same line if there is no other calls. Thoughts?